lib/pluginmanager/install.rb

```diff
@@ -45,9 +45,11 @@ def validate_cli_options!
   # Check if the specified gems contains
   # the logstash `metadata`
   def verify_remote!(gems)
+    options = { :rubygems_source => gemfile.gemset.sources }
     gems.each do |plugin, version|
       puts("Validating #{[plugin, version].compact.join("-")}")
-      signal_error("Installation aborted, verification failed for #{plugin} #{version}") unless LogStash::PluginManager.logstash_plugin?(plugin, version)
+      next if LogStash::PluginManager.logstash_plugin?(plugin, version, options)
+      signal_error("Installation aborted, verification failed for #{plugin} #{version}")
     end
```

Comment:
@purbon Can we wrap the call to logstash_plugins? with a begin / rescue clause? 
When I run `bin/plugin install my-plugin` without internet access and without `--no-verify` I get an exception. 

I suppose that if the user run his own gem server he may not have access to the internet.
I think we should provide a better error and not a stacktrace what do you think?


```
/e/logstash git:pr/3583 ❯❯❯ bin/plugin install logstash-mixin-rabbitmq_connection --no-verify                                                                                                                                                                                                                                                                          ⏎ ✱
Validating logstash-mixin-rabbitmq_connection
SocketError: recv: name or service not known
             recv_reply at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:782
                request at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:664
          each_resource at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:512
                 resolv at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:0
                   each at org/jruby/RubyArray.java:1613
                 resolv at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:1034
                   each at org/jruby/RubyArray.java:1613
                 resolv at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:1033
                   each at org/jruby/RubyArray.java:1613
                 resolv at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:1031
          each_resource at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:503
            getresource at /Users/ph/es/logstash/vendor/jruby/lib/ruby/1.9/resolv.rb:480
           api_endpoint at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/remote_fetcher.rb:92
                api_uri at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/source.rb:46
             load_specs at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/source.rb:182
             tuples_for at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/spec_fetcher.rb:261
        available_specs at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/spec_fetcher.rb:226
                   each at org/jruby/RubyArray.java:1613
            each_source at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/source_list.rb:97
        available_specs at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/spec_fetcher.rb:222
  search_for_dependency at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/spec_fetcher.rb:102
    spec_for_dependency at /Users/ph/es/logstash/vendor/jruby/lib/ruby/shared/rubygems/spec_fetcher.rb:166
       logstash_plugin? at /Users/ph/es/logstash/lib/pluginmanager/util.rb:25
         verify_remote! at /Users/ph/es/logstash/lib/pluginmanager/install.rb:51
                   each at org/jruby/RubyArray.java:1613
         verify_remote! at /Users/ph/es/logstash/lib/pluginmanager/install.rb:49
                execute at /Users/ph/es/logstash/lib/pluginmanager/install.rb:26
                    run at /Users/ph/es/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67
                execute at /Users/ph/es/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/subcommand/execution.rb:11
                    run at /Users/ph/es/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67
                    run at /Users/ph/es/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:132
                 (root) at /Users/ph/es/logstash/lib/pluginmanager/main.rb:37
```